README.md

```diff
@@ -6,6 +6,8 @@
 
 ## Prerequisites
 
+1. **Subscription**. In order to use the OpenShift on Azure plugin, your Azure subscription must be whitelisted to use the service. If you do not use a valid subscription, you will not have access to the appropriate resource groups in order to successfully run code in this repository. Reach out to the #aos-azure team in [Slack][slack] with any questions about this.
```

Comment:
I think the point I was wanting to get at is: these instructions are written on the basis that you're an aos-azure contributor with access to the subscription.  If you're not, these instructions are not going to get you all the way right now.

Reply:
Added a note along those lines above.